packages/ember-glimmer/lib/utils/references.js

```diff
@@ -52,6 +52,27 @@ export class UpdatableReference extends RootReference {
   }
 }
 
+export class GetHelperReference {
+  constructor(args) {
+    this.positionalArgs = args.positional;
+  }
+
+  isDirty() { return true; }
+
+  value() {
+    let objRef = this.positionalArgs.at(0).value(),
+      pathRef = this.positionalArgs.at(1).value();
+
+    return objRef ? get(objRef, pathRef) : objRef;
+  }
+
+  get(propertyKey) {
+    throw new Error("get() is not implemented");
```

Comment:
Doesn't seem to be a test for this. I would suggest adding a test that does this type of yielding.